frontendActions/src/main/resources/actions/ResourceBundle.properties

```diff
@@ -32,6 +36,14 @@ action.GitMachete.RebaseCurrentBranchOntoParentAction.text=Rebase Current Branch
 action.GitMachete.RebaseCurrentBranchOntoParentAction.GitMacheteToolbar.text=Rebase Current Branch Onto Parent
 action.GitMachete.RebaseCurrentBranchOntoParentAction.description=Rebase current branch onto parent
 
+action.GitMachete.FetchAllRemotesAction.text=Fetch All
```

Comment:
I'd specify `Fetch All Remotes` (it could also mean "fetch all branches (from one specific remote)")